web/src/search/input/toggles/Toggles.tsx

```diff
@@ -89,8 +91,19 @@ export const Toggles: React.FunctionComponent<TogglesProps> = (props: TogglesPro
         submitOnToggle({ newPatternType })
     }
 
+    const generateQuery = (): string =>
```

Comment:
Since this is always called immediately
```suggestion
    const fullQuery = 
```

Reply:
https://github.com/sourcegraph/sourcegraph/pull/10168/commits/84f8c0873af2480a03d93fd5c220168c380e0522